jobserver/models.py

```diff
@@ -362,17 +362,30 @@ class Project(models.Model):
         related_name="projects",
     )
 
-    name = models.TextField(unique=True)
-    email = models.TextField()
-    project_lead = models.TextField()
-    proposed_start_date = models.DateTimeField()
-    proposed_duration = models.TextField()
-
+    name = models.TextField(blank=True)
+    display_name = models.TextField(blank=True)
+    email = models.TextField(blank=True)
+    project_lead = models.TextField(blank=True)
+    proposed_start_date = models.DateTimeField(null=True, blank=True)
+    proposed_duration = models.TextField(blank=True)
+
+    # store a link to our modified FormA
+    form_url = models.TextField(blank=True)
+
+    # Governance approval
+    governance_approval_notes = models.TextField(blank=True)
     has_governance_approval = models.BooleanField(default=False)
+
+    # Technical approval
+    technical_approval_notes = models.TextField(blank=True)
     has_technical_approval = models.BooleanField(default=False)
 
+    # project published fields
+    paper_published_at = models.DateTimeField(null=True)
```

Comment:
It may not be a paper that's published; it could be a preprint or a notebook or a blog post, for example.

There may also be more than one associated publication (for example the Nature paper could be a preprint, a paper, and a blog post)

At the least I'd avoid the terminology `paper_` at this stage. Perhaps `primary_output_publication_url` (or `output_publication_urls` if you want to add one-to-many relationship now)?

There's also an option to ask people always to provide a DOI in this field; makes it much more likely everything is citable.

Reply:
Noted, this was added to try and start getting some quick and dirty metrics of projects created vs papers published.  I didn't realise multiple things could be linked though!  I'll remove for now as it's not something which was asked for.